Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+                persistent._mas_game_nou_house_rules["stack_d2"] = False
+                persistent._mas_game_nou_house_rules["play_wd4_anytime"] = False
+
+                persistent._mas_game_nou_points["Monika"] = 0
+                persistent._mas_game_nou_points["Player"] = 0
+
+                del[is_default_rules]
+                del[menu_items]
+                del[final_item]
+
+            return
+
+    $ persistent._mas_game_nou_points["Monika"] = 0
+    $ persistent._mas_game_nou_points["Player"] = 0
+
+    m 3eua "Is there anything else you would like to change?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Is there anything else you would like to change?{fast}"
+
+        "Yes.":
+            jump monika_nou_change_house_rules.selection_loop
+
+        "No.":
+            m 2eub "Then let's play together soon~"
+
+    python:
+        del[is_default_rules]
+        del[menu_items]
+        del[final_item]
+
+    return
+
+label .change_win_points_loop:
+    $ ready = False
+    while not ready:
+        show monika 1eua at t11
+
+        $ points_cap = store.mas_utils.tryparseint(
+            renpy.input(
+                "How many points would you like it to be?",
+                allow=numbers_only,
+                length=4
+            ).strip("\t\n\r"),
+            200
+        )
+
+        if points_cap < 0:
+            m 2rksdla "[player], how would we play with negative amount of point?"
+            m 3ekb "Try again, silly!"
+
+        elif points_cap == 0:
+            m 3eua "Oh, you just want to have quick games?"
+            m 2tuu "Alright! But don't expect me to go easy on you, though~"
+            $ persistent._mas_game_nou_house_rules["win_points"] = points_cap
+            $ ready = True
+
+        elif points_cap < 50:
+            m 3rksdlb "Hmm, It doesn't make sense to play with {i}that{/i} little amount of points."
+            m 1eka "We can play without points if you wish.{nw}"
+            $ _history_list.pop()
+            menu:
+                m "We can play without points if you wish.{fast}"
+
+                "I'd like that.":
+                    m 1eub "Oh, alrigh!"
+                    $ persistent._mas_game_nou_house_rules["win_points"] = 0
+                    $ ready = True
+
+                "No":
+                    m 3eua "Then choose again."
+
+        elif points_cap > 3000:
+            m 2eka "Oh it's too much I think..."
+            m 3eka "Let's leave it at 3000?{nw}"
+            $ _history_list.pop()
+            menu:
+                m "Let's leave it at 3000?{fast}"
+
+                "Alright.":
+                    m 1eua "Settled."
+                    $ persistent._mas_game_nou_house_rules["win_points"] = 3000
+                    $ ready = True
+
+                "No":
+                    m 3eua "Then choose again."
+
+        else:
+            m 3eub "Okay, from now wins the first who reaches [points_cap] points!"
```

Comment:
Grammar is wrong here due to word placement and missing words. Might I recommend "Okay, from now on, whoever reaches [points_cap] points, wins!"